platform/ios/src/MGLMapView.mm

```diff
@@ -230,6 +231,8 @@ @interface MGLMapView () <UIGestureRecognizerDelegate,
 @property (nonatomic) EAGLContext *context;
 @property (nonatomic) GLKView *glView;
 @property (nonatomic) UIImageView *glSnapshotView;
+@property (nonatomic, readwrite) MGLScaleBarView *scaleBarView;
```

Comment:
If we expose the view as an ordinary UIView, the developer can still hide the scale bar and we retain more flexibility to iterate on the view’s design and structure.